swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)
+        self.fabric = fabric
```

Comment:
Just a preference, haha! No need to make this change:
```suggestion
        self.fabric = fabric
        super().__init__(id, condition)
```

Perfect use of the `super` function!